pkg/delete/delete_test.go

```diff
@@ -843,3 +837,27 @@ func TestConfigsWithParent(t *testing.T) {
 		})
 	}
 }
+
+func Test_KeyUserActionsWebUniqueness(t *testing.T) {
```

Comment:
Should we add a few more test cases where we see specific KUA-web being deleted or _not_ deleted.